internal/cache/metadata/stat_cache_test.go

```diff
@@ -111,6 +113,7 @@ type MultiBucketStatCacheTest struct {
 func (t *StatCacheTest) SetupTest() {
 	cache := lru.NewCache(uint64((mount.AverageSizeOfPositiveStatCacheEntry + mount.AverageSizeOfNegativeStatCacheEntry) * capacity))
 	t.cache.wrapped = metadata.NewStatCacheBucketView(cache, "") // this demonstrates
+	t.statCache = metadata.NewStatCacheBucketView(cache, "")     // this demonstrates
```

Comment:
can't we use `t.cache` only? why did we need this separate variable?

Reply:
t.cache is the old wrapper class in tests that calls metadata.StatCache. This approach tests wrapper methods instead of actual functionality, compromising the safety net. If the helper class changes internally, tests won't fail. I've refactored this in my PR to test metadata.StatCache directly, removing unnecessary wrappers for accurate unit testing. Changing every test will increase the scope and actual hns work will be affected, so taking cautious call to just add new test in refactored way in first go, we can update rest of tests slowly later.

Reply:
Okay just write comment above it.